core/src/main/scala/kafka/tools/PerfConfig.scala

```diff
@@ -28,9 +28,9 @@ class PerfConfig(args: Array[String]) {
     .ofType(classOf[java.lang.Long])
   val reportingIntervalOpt = parser.accepts("reporting-interval", "Interval at which to print progress info.")
     .withRequiredArg
-    .describedAs("size")
+    .describedAs("length")
```

Comment:
Yes, thank you. Or "interval_ms" (I spotted similar usages elsewhere in the code)?